servicebroker/test/models/binding_test.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+var expect = require('chai').expect;
+var serviceInstance = require('../../lib/models')().service_instance;
+var binding = require('../../lib/models')().binding;
+
+var fake_serviceInstanceId = "test_serviceinstance";
```

Comment:
The naming is wrong.
These are not fakes but actual names.
Fakes are when you aren't using a real instance.